content/20.treatments.md

```diff
@@ -120,17 +120,52 @@ For that reason, neutralizing antibodies have emerged to address these shortcomi
+Each S glycoprotein protomer is comprised of an S1 domain also called the receptor binding domain (RBD), and an S2 domain.
+The S1 domain binds to host cell receptors while the S2 domain facilitates the fusion between the viral envelope and host cell membranes [@doi:10.1517/14712590902763755].
+Although targeting of the host cell receptor ACE2 shows efficacy in inhibiting SARS-CoV-2 infection [@doi:10.1038/nature02145], given the physiological relevance of ACE2 [@doi:10.1016/S1050-1738(02)00233-5], it would be favorable to target virus-specific structures rather than host receptors.
+This forms the rationale of developing neutralizing antibodies against the S glycoprotein, disrupting its interaction with ACE2 and other receptors and thereby inhibiting viral entry.
+
+
+###### Spike (S) Neutralizing Antibody Current Evidence
+
+The first human neutralizing antibody against SARS-CoV-2 targeting the trimeric spike (S) glycoproteins has been developed using hybridoma technology [@doi:10.1101/2020.03.11.987958].
+The 47D11 clone was able to cross-neutralize SARS-CoV and SARS-CoV2 by a mechanism that is different from receptor binding interference.
+The exact mechanism of how this clone neutralizes SARS-CoV-2 and inhibits infection in vitro remains unknown, but a potential mechanism might be antibody induced destabilization of the membrane prefusion structure[@doi:10.1101/2020.03.11.987958;@doi:10.1016/j.cell.2018.12.028].
+The ability of this antibody to prevent infection at a feasible dose needs to be validated in vivo, especially since in vitro neutralization effects have been shown to not be reflective of in vivo efficacy [@doi:10.1128/JVI.01603-17].
+Only a week later, a different group reported the isolation of multiple nAbs targeting the RBD of the S glycoprotein from COVID-19 patients [@doi:10.1101/2020.03.21.990770].
+Interestingly, the patient isolated antibodies did not cross-react with RBD’s from SARS-CoV and MERS-CoV, although cross-reactivity to the trimeric spike proteins of SARS-CoV and MERS-CoV was observed.
+This suggests that the RBD’s between the three coronavirus species are immunologically distinct and that the isolated nAbs targeting the RBD of SARS-CoV-2 is species specific.
+While specificity is desirable, it also poses the question of whether these antibodies are more susceptible to viral escape mechanisms.
+Viral escape is a common resistance mechanism to nAbs therapy due to selective pressure from neutralizing antibpdies [@doi:10.2217/imt.15.33; 10.1126/science.1213256].
+For HIV, broadly neutralizing antibodies (bnAbs) targeting the CD4bs have showed greater neutralization breadth than normal monoclonal antibodies [@doi:10.1016/j.molmed.2019.01.007].
+For MERS-CoV, a combination of multiple neutralizing antibodies targeting different antigenic sites were able to avoid neutralization escape [@doi:10.1128/JVI.02002-17].
```

Comment:
```suggestion
For MERS-CoV, a combination of multiple neutralizing antibodies targeting different antigenic sites prevented neutralization escape [@doi:10.1128/JVI.02002-17].
```
"Avoid" sounds a little weird to me (it feels like it's implying the drugs have agency I guess?), so I'm suggesting a phrase they use in the paper (they say "combining MAbs targeting distinct epitopes prevented viral neutralization escape from RBD-directed MAbs.")